modules/deployments-ab-testing-lb.adoc

```diff
@@ -113,23 +113,36 @@ spec:
 ----
 
 [id="deployments-ab-testing-lb-web_{context}"]
-== Managing weights using the web console
+== Managing weights of an existing route using the web console
 
 .Procedure
 
-. Navigate to the Route details page (Applications/Routes).
+. Navigate to the *Networking -> *Routes* details page.
 
-. Select *Edit* from the Actions menu.
+. Click the Options menu {kebab} next to the Route you want to edit and select *Edit Route*.
```

Comment:
If a user is on Route details page, no Options menu, there is only Actions menu, see below screenshot
<img width="1386" alt="Screen Shot 2020-11-05 at 9 49 33 AM" src="https://user-images.githubusercontent.com/12692381/98187781-aa4bc500-1f4c-11eb-89c8-ea2569f82b90.png">

If a user is on Routes list page, he can see Options menu, see below screenshot
<img width="1397" alt="Screen Shot 2020-11-05 at 9 51 02 AM" src="https://user-images.githubusercontent.com/12692381/98187793-b172d300-1f4c-11eb-813b-7e8f86e9b25d.png">

Reply:
Also it seems an extra * in preview build
<img width="624" alt="Screen Shot 2020-11-05 at 9 53 45 AM" src="https://user-images.githubusercontent.com/12692381/98187866-dbc49080-1f4c-11eb-9b8c-b173d54ce7a2.png">